src/plots/plot.ts

```diff
@@ -855,6 +865,41 @@ export class Plot extends Component {
     return nearest === undefined ? undefined : this._lightweightPlotEntityToPlotEntity(nearest);
   }
 
+  /**
+   * Returns the entites whose bounding boxes overlap the parameter.
+   */
+  public entitiesInBounds(queryBounds: IEntityBounds): Plots.IPlotEntity[] {
```

Comment:
Nit: to be clear - this ignores the `y/height` properties on IEntityBounds? This API feels inconsistent with the existing `entitiesIn(xRangeOrBounds: Range | Bounds, yRange?: Range): IPlotEntity[] {` that bar, scatter, line, and rectangle have.

I think part of the issue is that we have overlapping responsibilities in our interfaces:

```ts
export type Range = {
  min: number;
  max: number;
};

/**
 * A location in pixel-space.
 */
export type Point = {
  x: number;
  y: number;
};

/**
 * The corners of a box.
 */
export type Bounds = {
  topLeft: Point;
  bottomRight: Point;
};

/**
 * Client bounds
 */
export interface IEntityBounds {
  x: number;
  y: number;
  width: number;
  height: number;
}
```

An `IEntityBounds` and a `Bounds` are semantically equivalent to each other, as well as equivalent to two `Range`s.

Reply:
very true. i'm not sure why the original bounds was structured as two points, but i've always found it awkward.

Reply:
I'll take a look at replacing the existing entitiesIn methods to defer to the relevant entitystore method